eng/DotNetBuild.props

```diff
@@ -112,13 +112,14 @@
       <InnerBuildArgs Condition="'$(GitHubRepositoryName)' != ''">$(InnerBuildArgs) /p:GitHubRepositoryName=$(GitHubRepositoryName)</InnerBuildArgs>
 
       <!-- Handle system libraries -->
-      <UseSystemLibs Condition="'$(UseSystemLibs)' != ''">+$(UseSystemLibs)+</UseSystemLibs>
-      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('+brotli+'))">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_BROTLI=true</InnerBuildArgs>
-      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('+libunwind+'))">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_LIBUNWIND=true</InnerBuildArgs>
+      <UseSystemLibs Condition="'$(UseSystemLibs)' != ''">+$(UseSystemLibs)</UseSystemLibs>
+      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('+brotli'))">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_BROTLI=true</InnerBuildArgs>
+      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('+libunwind'))">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_LIBUNWIND=true</InnerBuildArgs>
       <!-- TODO: llvm-libunwind -->
       <!-- TODO: LinuxTracepoints -->
-      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('+rapidjson+'))">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_RAPIDJSON=true</InnerBuildArgs>
-      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('+zlib+'))">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_ZLIB=true</InnerBuildArgs>
+      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('+rapidjson'))">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_RAPIDJSON=true</InnerBuildArgs>
+      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('+zlib'))">$(InnerBuildArgs) --cmakeargs -DCLR_CMAKE_USE_SYSTEM_ZLIB=true</InnerBuildArgs>
+      <InnerBuildArgs Condition="$(UseSystemLibs.Contains('-lttng'))">$(InnerBuildArgs) /p:FeatureXplatEventSource=false</InnerBuildArgs>
```

Comment:
For using this from the vmr `build.sh`, I'm using the existing `--with-system-libs` argument. This arguments allows combining libraries with a `+`. I use `-` to mean _minus/subtract_.

The semantics do not match 100%, but it avoids the work of introducing and wiring up another property.

Reply:
Seems like `UseSystemLibs` has two meanings:
* on: use system lib, off: use source build using in-tree code
* include/exclude libs

Should we split it in two props or keep the FeatureXYZ thing separate without dealing with UseSystemLibs?

Reply:
I'd like to add it here using `UseSystemLibs` because that is available from the vmr `build.sh`.

If desired, we can wire up something from the other end and then replace it here.

Reply:
Makes sense. In a long run, I think the feature switches are probably something worth a discussion for VMR in general; `--with(out)-feature-xyz` which would map to Feature_XYZ=1/0 in concerned repos.

e.g. some native libs are not optional (like libunwind), so they won't have a CLI arg.

Reply:
@ViktorHofer @jkoritzinsky Are you ok with overloading `--with-system-libs` to disable runtime features?

Reply:
I'd rather have a `--with-feature-xyz` or a `--no-feature-xyz` UX. I'm trying to change how we consume possibly-system libraries in #113176 and this would make the implementation significantly more confusing.

Reply:
@jkoritzinsky are you ok with using `UseSystemLibs` for this PR?

> and this would make the implementation significantly more confusing.

Probably the handling of `-lttng` could be localized to `eng/DotNetBuild.props`?

Reply:
We can do it for this PR, sure. If you can localize the `-lttng` behavior to `eng/DotNetBuild.props` I'd appreciate it.

Reply:
Here it is already localized to `eng/DotNetBuild.props`. That should be possible in https://github.com/dotnet/runtime/pull/113176 too.